plugins/com.google.cloud.tools.eclipse.swtbot/src/com/google/cloud/tools/eclipse/swtbot/SwtBotProjectActions.java

```diff
@@ -249,6 +251,18 @@ public static SWTBotTreeItem selectProjectItem(SWTBotTreeItem item, String... fo
     return item;
   }
 
+  /** Collapse all projects shown in the Project Explorer. */
```

Comment:
I guess not, but is there any chance it will be the Package Explorer in our tests?

Reply:
Most of our tests do have the _Package Explorer_ open, but the _Package Explorer_ doesn't support extension and so this `IModelProvider` log traces don't apply.